score_email.cwl

```diff
@@ -54,7 +54,12 @@ requirements:
           syn.login()
 
           sub = syn.getSubmission(args.submissionid)
-          userid = sub.userId
+          participantid = sub.get("teamId")
+          if teamid is not None:
```

Comment:
Where do you assign `teamid` ? Did you test this?

Reply:
Good catch, I have now fixed and tested.  It was untested at the time of your comment